spec-main/api-tray-spec.ts

```diff
@@ -1,18 +1,20 @@
-const { remote } = require('electron')
-const { expect } = require('chai')
-const { Menu, Tray, nativeImage } = remote
+import { expect } from 'chai'
+import { Menu, Tray, nativeImage } from 'electron'
 
 describe('tray module', () => {
-  let tray
+  let tray: Tray;
 
   beforeEach(() => {
     tray = new Tray(nativeImage.createEmpty())
   })
 
+  afterEach(() => {
+    tray = null as any
```

Comment:
The pattern I use elsewhere is
```suggestion
    tray = null as unknown as Tray
```

unclear to me if that's any better though? 🤷‍♂

Reply:
hmm apart from the fact that it looks more complicated... probably doesn't matter

Reply:
I think in other places I just used `as any` to force this null resetting 👍